engine/api/project.go

```diff
@@ -631,12 +630,12 @@ func (api *API) deleteProjectHandler() service.Handler {
 		vars := mux.Vars(r)
 		key := vars[permProjectKey]
 
-		p, errProj := project.Load(api.mustDB(), api.Cache, key, project.LoadOptions.WithPipelines, project.LoadOptions.WithApplications)
-		if errProj != nil {
-			if !sdk.ErrorIs(errProj, sdk.ErrNoProject) {
-				return sdk.WrapError(errProj, "deleteProject> load project '%s' from db", key)
+		p, err := project.Load(api.mustDB(), key, project.LoadOptions.WithPipelines, project.LoadOptions.WithApplications)
+		if err != nil {
+			if !sdk.ErrorIs(err, sdk.ErrNoProject) {
```

Comment:
no really useful to preserve this if?

Reply:
probably not, but o have to work on the project DAO in a next PR